src/main/webapp/assets/styles/components/bsp-gallery-fullscreen.less

```diff
@@ -217,3 +260,21 @@ body.bsp-gallery-fullscreen {
     }
 
 }
+
```

Comment:
Can you put the below CSS here: https://github.com/perfectsense/brightspot-base/blob/master/src/main/webapp/assets/styles/components/move-to-bsp-carousel.less